File: data-structure-and-algorithm/java/src/com/bestlyg/utils/model/Patient.java
package com.bestlyg.utils.model;

public class Patient implements Comparable<Patient> {
	private String name;
	private int boneBreak;

	public Patient(String name, int boneBreak) {
		this.name = name;
		this.boneBreak = boneBreak;
	}

	@Override
	public int compareTo(Patient patient) {
		return this.boneBreak - patient.boneBreak;
	}

	@Override
	public String toString() {
		return "Person [name=" + name + ", boneBreak=" + boneBreak + "]";
	}
}

File: data-structure-and-algorithm/typescript/src/utils/model/Patient.ts
export default class Patient {
  private _name: string;
  private _boneBreak: number;
  constructor(name: string, boneBreak: number) {
    this._name = name;
    this._boneBreak = boneBreak;
  }
  get name(): string {
    return this._name;
  }
  get boneBreak(): number {
    return this._boneBreak;
  }
  toString(): string {
    return `Patient name:${this._name} boneBreak:${this._boneBreak}`;
  }
}
